examples/unattended_wireguard_example.conf

```diff
@@ -9,4 +9,5 @@ pivpnPORT=51820
 pivpnDNS1=9.9.9.9
 pivpnDNS2=149.112.112.112
 pivpnHOST=pivpn.example.com
+pivpnPERSISTENTKEEPALIVE=21
```

Comment:
WireGuard website suggests 25 as default value: https://www.wireguard.com/quickstart/#nat-and-firewall-traversal-persistence

Reply:
That might be and I also update to the suggested value but 21 is the value that we found working better than the 25 seconds. However I understand that we should probably go for the suggested value.